### PR TITLE
Added Concise Toggle

### PR DESCRIPTION
**Pull request by aravind-segu**
#31 

Added a Concise Version Toggle in the Table of Contents

image

Currently Excluded the Following Sections from the Concise Version:
- Demographics and Background
    - Year of Birth
    - Immigrated to Canada
    - Polyglots
    - Number of Siblings
    - CEGEP
- Academics
    - Options
    - Transferred From
    - Overloading
    - Location
- Coop
    - Favorite Locations
- Lifestyle & Experiences
    - Favourite Mobile US
    - Average Sleep Duration
    - Design Teams
- Health
    - Insurance Coverage
    - Intermurals
    - Change in Weight
- Family and Relationships
    - Cheating
- Finances
    - Expenses on School Material
- Exchange
- Misc
    - Physical Conflict
    - Cat or Dog
    - Completing Bachelors 
- Transfers
    - Switch Years
    - Regret
- Outcome
    - Experience Per Gender
- Future
    - Professional Engineers

Feel Free to comment on if I should exclude more or should any of the above sections. I tried to keep the stories in order to make it story based. 

To Hide a section surround the html content with `<div class="long_version>...</div>`

### CODE REVIEW

File: content/introduction.html [hunk 1]
```diff
@@ -152,3 +152,11 @@ <h1 id="toc">Sections</h1>
     </li>
   </ul>
 </div>
+<div class="vbx2"></div>
```
Comment (roxelo) [inline content/introduction.html:155]: Maybe add a short explanation of what the concise toggle is

**Review by roxelo — COMMENTED** [review]
This looks awesome!!!!!

There are a few other graphs which I think should not be included in the concise version:

- [x] Enriched Programs vs Grades
- [x] Sleep and Grades
- [x] Interviews Section
- [x] Hackathons vs Coop Pay
- [x] Side Project vs Coop Pay
- [x] Moving Home
- [ ] Cooking Habits
- [x] Crying Frequency
- [x] Friendship Lost and Gained
- [ ] Time Spent with Family

**Comment by roxelo** [comment]
LGTM

**Comment by AkshayPall** [comment]
Thanks for taking this up! It looks good - I just want to remove a few other things also

- [ ] Grocery store questions
- [ ] Dining questions
- [ ] Burnout
- [ ] Reddit usage

File: content/introduction.html [hunk 2]
```diff
@@ -147,8 +147,20 @@ <h1 id="toc">Sections</h1>
...
+<div>
+  <h3 id="concise">Concise Version</h3>
+  <label class="switch">
+    <input id="concise-toggle" type="checkbox">
+    <span class="slider round"></span>
+  </label>
+  <p>
```
Comment (jaxonL) [inline content/introduction.html]: unsure why/if it was intentional, but the preview shows the description of the toggle like this:

image

maybe we can center-align the text in the description? (or move this to a sass file)
```suggestion
  <p style="text-align:center;">
```
image

**Review by jaxonL — COMMENTED** [review]
looks good 📏 left a comment about the description text alignment

**Review by roxelo — APPROVED** [review]

**Pull request merged**